### PR TITLE
Overhaul the multiplayer menu UI

### PR DESCRIPTION
**Pull request by pchote**
The discussion and subsequent removal of the ingame IRC client raised some points that we need to finish addressing before the playtest. This PR makes a start by addressing two points:
* The tabbed layout (and size/shape constraints that imposed) doesn't make sense without the chat to tie them together.
* Many players don't understand port forwarding.

This effectively reverts #9744.

screen shot 2017-12-22 at 14 48 43
screen shot 2017-12-22 at 16 52 56
screen shot 2017-12-22 at 16 53 46
screen shot 2017-12-22 at 17 01 49
screen shot 2017-12-22 at 16 53 15

### CODE REVIEW

**Comment by pchote** [comment]
Once this is merged a followup PR will add a player list to the MP browser, which is the main motivation for this exercise:
screen shot 2017-12-22 at 16 44 17

File: OpenRA.Game/Game.cs [hunk 1]
```diff
@@ -872,7 +866,6 @@ public static int CreateLocalServer(string map)
 				Name = "Skirmish Game",
 				Map = map,
 				AdvertiseOnline = false,
```
Comment (reaperrr) [inline OpenRA.Game/Game.cs]: Minor style nit: now that it's the last entry, the comma could go away.

**Review by reaperrr — DISMISSED** [review]
Layout looks good to me, all dialogs seems to work as intended, mod switching still works

**Comment by pchote** [comment]
Updated.

**Review by abcdefg30 — APPROVED** [review]

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/10722c822c08ca0b86b5e565870f84b7d4ba2f22...d924a472f79aff72163d6c2d5f088298b168ad41)

**Pull request merged**